microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -38,227 +56,269 @@ To use experimental APIs, download a prerelease version of the WebView2 SDK from
 To download a Microsoft Edge preview channel, see [Microsoft Edge Insider Channels](https://www.microsoft.com/edge/download/insider).
 
 
-<!-- intro/overview of 4 approaches ======================================= -->
+<!-- ====================================================================== -->
 ## Approaches to making your app use a specific browser channel
 
-When a WebView2 is initialized, it will attempt to find a valid runtime on the machine to use. This can be the WebView2 Runtime, a preview channel of Microsoft Edge, or a specified location containing fixed version binaries. You can learn more about supported runtimes at [Distribute your app and the WebView2 Runtime](../concepts/distribution.md).
+When a WebView2 is initialized, it will attempt to find a valid runtime on the machine to use. This can be the WebView2 Runtime, a preview channel of Microsoft Edge, or a specified location containing Fixed Version binaries.  For more information about supported runtimes, see [Distribute your app and the WebView2 Runtime](../concepts/distribution.md).
 
-There are several ways to make your WebView2 app use a specified preview channel of Microsoft Edge:
-*  By calling a function.
-*  By using a registry override.
-*  By using an environment variable.
-*  By using a group policy.
+There are two ways to make your WebView2 app use a specified preview channel of Microsoft Edge:
 
-These approaches are described below.
+* [Switching the channel search order (recommended)](#switching-the-channel-search-order-recommended).
 
-### Browser executable folder
+* [Setting the browser executable folder (for local testing)](#setting-the-browser-executable-folder-for-local-testing).
 
-One approach is to use a browser executable folder.  In this approach, you specify a folder that contains runtime binaries. This folder can be any of the following locations:
-*  The installed location of the WebView2 Runtime.
-*  A preview channel of Microsoft Edge.
-*  A folder containing fixed version binaries that you have deployed to the machine yourself.
+These approaches are described below:
 
-If you set the browser executable folder to a specific preview channel of Microsoft Edge, you will need to update the location when that preview channel updates to a newer version. This is because the location includes the version number as part of its path. Therefore, we recommend using this approach for local testing only.
 
-### Default channel-search order
+<!-- ====================================================================== -->
+## Switching the channel search order (recommended)
```

Comment:
This is also like the third-time we are describing how to do this. We really should just have 1 place properly describe this, and then link to it from the other locations. I recommend this doc be the source of truth, and the other testing docs link here.

Reply:
deduped the information